Assets/Mirage/Editor/WelcomeWindow/WelcomeWindow.cs

```diff
@@ -132,7 +135,14 @@ private void OnEnable()
             ConfigureTab("SponsorButton", "Sponsor", SponsorUrl);
             ConfigureTab("DiscordButton", "Discord", DiscordInviteUrl);
 
-            ShowTab(ShowChangeLog ? "ChangeLog" : "Welcome");
+            ShowTab(EditorPrefs.GetString(screenToOpenKey, "Welcome"));
+
+            //set the screen's button to be tinted when welcome window is opened
+            float color = EditorPrefs.GetFloat("buttonClickedColor");
+            float borderColor = EditorPrefs.GetFloat("buttonClickedBorderColor");
+            Button openedButton = rootVisualElement.Q<Button>(EditorPrefs.GetString(screenToOpenKey, "Welcome") + "Button");
+            openedButton.style.backgroundColor = new StyleColor(new Color(color, color, color));
```

Comment:
a better way is to define a uss class,  like this
```css
.selected-tab {
    background-color = ...
 }
 ```
 
 Then in code,  you can add or remove this class like this:
```cs
// enable it by passing true
openedButton.EnableInClassList("selected-tab", true);
// disable it by passing false
openedButton.EnableInClassList("selected-tab", false);
```
 
This way we are able to change any style for the button,  not just color, and we keep presentation in the xml/uss and logic in C#

If you need it,  here is a list of all the uss properties supported:
https://docs.unity3d.com/Manual/UIE-USS-SupportedProperties.html